test/parallel/test-cluster-process-disconnect.js

```diff
@@ -6,8 +6,8 @@ const cluster = require('cluster');
 if (cluster.isMaster) {
   const worker = cluster.fork();
   worker.on('exit', common.mustCall((code, signal) => {
-    assert.strictEqual(code, 0, 'worker did not exit normally');
-    assert.strictEqual(signal, null, 'worker did not exit normally');
+    assert.strictEqual(code, 0, `Worker did not exit normally with code: ${code}`);
+    assert.strictEqual(signal, null, `Worker did not exit normally with signal: ${signal}`);
```

Comment:
The line length exceeds 80 characters and has to be fixed. This could be done while landing as well though.

Reply:
Thanks @lance @BridgeAR , I have updated the commit message to reflect the limit, my apologies. Please let me know if there is anything else.

Reply:
It would have been great if you would have also changed this to reflect the line length ^^